media-gfx/opentoonz/opentoonz-1.2.0.recipe

```diff
@@ -1,23 +1,23 @@
 SUMMARY="An open-source full-featured 2D animation creation software"
 DESCRIPTION="OpenToonz is a 2D animation software published by \
-DWANGO (http://dwango.co.jp/english/). It is based on Toonz Studio \
-Ghibli Version, originally developed in Italy by \
-Digital Video, Inc. (http://www.toonz.com/), and customized by \
-Studio Ghibli (http://www.ghibli.jp/) over many years of production."
-LICENSE="BSD (3-clause)"
-COPYRIGHT="2016, DWANGO Co., Ltd."
+DWANGO. It is based on Toonz Studio Ghibli Version, originally developed \
+in Italy by Digital Video, Inc., and customized by Studio Ghibli over \
+many years of production."
 HOMEPAGE="https://github.com/opentoonz/opentoonz/"
+COPYRIGHT="2016 DWANGO Co., Ltd."
+LICENSE="BSD (3-clause)"
 REVISION="1"
-SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="e9943abbd49befb5c8963fd7cb744b6c5b93a4faa02c8564610b848d189ea029"
+SOURCE_URI="https://github.com/opentoonz/opentoonz/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="8aca2f257f07d4a5037ed9ccc2d007f941b7813a7ec6ca92c0e2ae21a43f8c96"
 PATCHES="opentoonz-$portVersion.patchset"
 
-ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="?x86"
 
 PROVIDES="
 	opentoonz$secondaryArchSuffix = $portVersion
-	cmd:opentoonz_1.1
+	cmd:lzocompress
+	cmd:lzodecompress
 	cmd:opentoonz
 	cmd:tcleanup
 	cmd:tcomposer
```

Comment:
Since everything is placed in `appsDir`, shouldn't we replace all `cmd:`s with `app:`s?